lib/events.js

```diff
@@ -306,7 +306,8 @@ EventEmitter.prototype.removeListener =
         else {
           delete events[type];
           if (events.removeListener)
-            this.emit('removeListener', type, listener);
+            this.emit('removeListener', type,
```

Comment:
Jeeze, has it been 6 days? Apologies.

Had a few long days at work and needed the weekend to unwind. This is in my inbox to get to - probably tomorrow night.